### PR TITLE
fix: centerize a string in a reporter when adding a padding

### LINKED ISSUE #330: (i) has too large padding
This issue was introduced in 472f3f3c4220b2a8c1de849ef784383ee4266157

image

### PR DESCRIPTION
**Pull request by exKAZUu**
Close #330

image

### CODE REVIEW

File: scratch3/blocks.js [hunk 1]
```diff
@@ -518,6 +519,7 @@ BlockView.prototype.draw = function() {
         : this.isCommand || this.isOutline ? 64 : this.isReporter ? 48 : 0,
     innerWidth
   )
+  padLeft += (innerWidth - orgInnerWidth) / 2
```
Comment (tjvr) [inline scratch3/blocks.js:522]: Probably worth a comment:
```suggestion
  // Center the label text inside small reporters.
  padLeft += (innerWidth - orgInnerWidth) / 2
```

File: scratch3/blocks.js [hunk 2]
```diff
@@ -510,6 +510,7 @@ BlockView.prototype.draw = function() {
   // Commands have a minimum width
   // The hat min-width is arbitrary (not sure of Scratch 3 value)
   // Outline min-width is deliberately higher (because Scratch 3 looks silly)
+  var orgInnerWidth = innerWidth
```
Comment (tjvr) [inline scratch3/blocks.js:513]: What's `org` short for? Original? Maybe `originalInnerWidth` would be better :-)

**Comment by exKAZUu** [comment]
Thank you for the review! I've fixed the code.

**Comment by tjvr** [comment]
Thanks!

**Pull request merged**